dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/K8sNamespaceMapper.xml

```diff
@@ -45,4 +50,38 @@
         </if>
     </select>
 
+    <select id="queryNamespaceExceptUserId" resultType="org.apache.dolphinscheduler.dao.entity.K8sNamespace">
+        select
+        <include refid="baseSql"/>
+        from t_ds_k8s_namespace
+        where user_id <![CDATA[ <> ]]> #{userId}
+    </select>
+
+    <select id="queryAuthedNamespaceListByUserId" resultType="org.apache.dolphinscheduler.dao.entity.K8sNamespace">
+        select
+        <include refid="baseSqlV2">
+            <property name="alias" value="p"/>
+        </include>
+        from t_ds_k8s_namespace p,t_ds_relation_namespace_user rel
+        where p.id = rel.namespace_id and rel.user_id= #{userId}
+    </select>
+
+    <select id="queryNamespaceAvailable" resultType="org.apache.dolphinscheduler.dao.entity.K8sNamespace">

```

Comment:
```suggestion
select 
<include refid="baseSqlV2">
   <property name="alias" value="b"/>
</include>
from t_ds_k8s_namespace ns 
left join t_ds_relation_namespace_user u on u.namespace_id = ns.id
where 1 = 1 and 
<if "userId !=null and userId != ''>
   u.user_id = #{userId}
</if>
```

Reply:
I think here need not to change, because  if userId is null,  t_ds_relation_namespace_user need not join